doc/compiler/extensible_classes/Extensible_Classes_in_OMR.md

```diff
@@ -35,8 +35,12 @@ architectures. These are also assigned namespaces as follows:
   - `i386`: for 32-bit x86 (x86-32)
 - `Z`: for the IBM z Systems architecture
 
-Additionally, there is a `TR` namespace, used by all projects. This namespace
-holds all concrete classes.
+In order to provide a concrete name that's usable anywhere, we map all our
```

Comment:
There probably isn't a good "verb" to describe what we're doing there, given we subclass the most derived class in the TR namespace. I'm not recommending you use that phrase, necessarily, but I'm worried that few people will associate "export" (or "map") with that action of subclassing in TR. Since we're trying to make it clear how people should use the TR namespace, I think it's worth an extra sentence or two to make our common (but loose) use of terminology more clear.

Thanks for your efforts to clear this up, @mgaudet !